components/SpotCard.js

```diff
@@ -0,0 +1,48 @@
+import styled from "styled-components";
+import Image from "next/image";
+import placeholderSpotImage from "../assets/images/card-image-placeholder.jpg";
+
+export default function SpotCard(spot) {
+  return (
+    <>
+      <Card>
+        <SpotImage src={placeholderSpotImage} alt="Placeholder Spot Image" />
+        <SpotInfos>
+          <h3>{spot.spot.category}</h3>
+          <h4>"{spot.spot.name}"</h4>
+        </SpotInfos>
+      </Card>
+    </>
+  );
+}
+
+const Card = styled.li`
+  background-color: white;
+  border: 1px solid black;
+  border-radius: 10px;
+  margin: 20px 20px 0px -20px;
```

Comment:
Negative `margin` and `padding` values are always a kind of weird and in most cases we can solve it differently.
See my comment in the `pages/index.js`. You try to get rid of the default `<ul>` style by setting `-20px` to the left - just set the `<ul>` `padding` to `0` and the `<li>` `margin` to `20px`. 🤯

```suggestion
  margin: 20px;
```